Modelica/Magnetic/FluxTubes/Basic/Short.mo

```diff
@@ -1,9 +1,6 @@
 within Modelica.Magnetic.FluxTubes.Basic;
 model Short "Short cut branch"
   extends Interfaces.TwoPortsElementary;
-equation
-  connect(port_p, port_n) annotation (Line(points={{-100,0},{-1,0},{-1,0},{
-          100,0}}, color={255,128,0}));
```

Comment:
This seems to remove the short from the Short-model. It both seems unrelated and wrong.

Reply:
Re-introduced by dcb71bc74fd7f67a48d8dde6b32cdc49af9ed1e9